db/migrate/20181016140921_migrate_orch_stacks_to_have_ownership_concept.rb

```diff
@@ -0,0 +1,85 @@
+class MigrateOrchStacksToHaveOwnershipConcept < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+
+    def tenant_identity
+      User.super_admin
+    end
+  end
+
+  class Service < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+
+    belongs_to :evm_owner, :class_name => "MigrateOrchStacksToHaveOwnershipConcept::User"
+
+    def tenant_identity
+      user = evm_owner
+      user = User.super_admin if user.nil? || !user.miq_group_ids.include?(miq_group_id)
+      user
+    end
+  end
+
+  class MiqGroup < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+
+    belongs_to :tenant, :class_name => "MigrateOrchStacksToHaveOwnershipConcept::Tenant"
+  end
+
+  class User < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+
+    has_and_belongs_to_many :miq_groups, :class_name => "MigrateOrchStacksToHaveOwnershipConcept::MiqGroup"
+    belongs_to :current_group,           :class_name => "MigrateOrchStacksToHaveOwnershipConcept::MiqGroup"
+
+    include ActiveRecord::IdRegions
+
+    def self.super_admin
+      find_by(:userid => "admin")
```

Comment:
Is `in_my_region` available?

```suggestion
          in_my_region.find_by_userid("admin")
```